### PR TITLE
feature_feedback_button

### PR DESCRIPTION
**Pull request by EvelienSG**
Please review my button and other changes.

### CODE REVIEW

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java [hunk 1]
```diff
@@ -31,5 +32,20 @@ public void onClick(View v) {
...
         });
+
+        final Button button2 = (Button) findViewById(R.id.buttonFeedback);
+        button2.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                //Start send email to specific address activity
+                sendMail();
+            }
+
+            private void sendMail() {
```
Comment (wpbs) [inline app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java]: Please add a check here for the possibility to handle email intents, like in: https://github.com/devolksbank/NL-Help-U/blob/fca4c95a366956c0ba10b09a83416715ffd8929b/app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentCollectionActivity.java#L253-L257 and also please check that the content associated to it matches this action: R.string.mail_handler_error

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java [hunk 2]
```diff
@@ -31,5 +32,20 @@ public void onClick(View v) {
                 startActivity(i);
             }
         });
+
+        final Button button2 = (Button) findViewById(R.id.buttonFeedback);
```
Comment (wpbs) [inline app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java]: missing update to the resource file associated to this activity, containing: R.id.buttonFeedback

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java [hunk 3]
```diff
@@ -12,7 +12,8 @@
 
 public class AboutActivity extends AppCompatActivity {
```
Comment (wpbs) [inline app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java]: Missing textual changes for the GitHub link / https://github.com/devolksbank/NL-Help-U/projects/1#card-5156629

Comment (EvelienSG) [reply app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java]: Please look at About page, 3rd text section. It states: "Are you interested in contributing to this app? Please go to thhps://github.com/devolksbank/NL-Help-U. Or contact the developers by using the contact button." Did you expect something else? Or a different page or section?

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java (hunk 2 above)
Comment (EvelienSG) [reply app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java]: Thanks, will do.

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java (hunk 3 above)
Comment (wpbs) [reply app/src/main/java/nl/devolksbank/nlhelpu/activity/AboutActivity.java]: the text is fine, but the update to the resource file has not been committed/pushed I guess ;)

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentCollectionActivity.java [hunk 4]
```diff
@@ -234,24 +235,13 @@ private static void copyFile(File src, File dst) throws IOException {
...
+
 
     private void sendMail() {
         Log.d("DocCollectionActivity", "Sending mail");
 
         Intent emailIntent = new Intent(Intent.ACTION_SEND_MULTIPLE);
-        if (!isIntentHandlerPresent(emailIntent)) {
+        if (!IntentUtil.isIntentHandlerPresent(emailIntent, getApplicationContext())) {
             Log.w("DocCollectionActivity", "No app present for handling an email intent");
             Toast.makeText(getApplicationContext(), getString(R.string.mail_handler_error), Toast.LENGTH_LONG).show();
```
Comment (wpbs) [inline app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentCollectionActivity.java]: This string now contains:
`<string name="mail_handler_error">No email app found on the device to send the documents</string>`
so please update the text to a more generic version usable in both cases

**Pull request merged**